chapters/annotations.tex

```diff
@@ -383,20 +383,25 @@ \section{Annotations for Simulations}\label{annotations-for-simulations}
 \end{center}
 
 \begin{annotationdefinition}[experiment]
-% henrikt-ma 2021-04: Seems strange to allow 'experiment' completely without list of options -- what would it mean?
 \begin{synopsis}[grammar]\begin{lstlisting}
-"experiment"
-   [ "(" [ experimentOption { "," experimentOption } ] ")" ]
-
-experimentOption:
-   "StartTime" "=" [ "+" | "-" ] UNSIGNED-NUMBER
-   | "StopTime" "=" [ "+" | "-" ] UNSIGNED-NUMBER
-   | "Interval" "=" UNSIGNED-NUMBER
-   | "Tolerance" "=" UNSIGNED-NUMBER
+annotation "experiment" "(" experimentModifiers ")"
 \end{lstlisting}\end{synopsis}
 \begin{semantics}
-The \lstinline{experiment} annotation defines the default start time (\lstinline!StartTime!) in {[}s{]}, the default stop time (\lstinline!StopTime!) in {[}s{]}, the suitable time resolution for the result grid (\lstinline!Interval!) in {[}s{]}, and the default relative integration tolerance (\lstinline!Tolerance!) for simulation experiments to be carried out with the model or block at hand.
+The modifiers are for the following record.
+\begin{lstlisting}[language=modelica]
+record experiment
+  Real StartTime(unit = "s") = 0;
+  Real StopTime(unit = "s");
+  Real Interval(unit = "s") = $\ldots{}$;
+  Real Tolerance = $\ldots{}$;
+end experiment;
```

Comment:
Let's try not to introduce a new way of mixing grammar style presentation with pseudo-code records.  Just using the record style of presentation would be better, combined with clarifications in the text below as needed.